cli/parser.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2020 Sebastian Pipping <sebastian@pipping.org>
+ * Licensed under AGPL v3 or later
+ */
+package cli
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/hartwork/go-wait-for-it/syntax"
+	"github.com/spf13/cobra"
+)
+
+type Config struct {
+	Addresses []syntax.Address
+	Argv      []string
+	Quiet     bool
+	Timeout   time.Duration
+}
+
+func report(err error) {
+	fmt.Println("Error:", err)
+}
```

Comment:
I wouldn't create this extra func, just inline it for these two occasions.

Reply:
I want to have the two places in sync at all times so I think I'll stick to D.R.Y. here: The added complexity seems small enough to me for that to be okay.